packages/master-detail-layout/src/vaadin-master-detail-layout.js

```diff
@@ -349,7 +346,8 @@ class MasterDetailLayout extends SlotStylesMixin(ResizeMixin(ElementMixin(Themab
 
       /**
        * When true, the component uses the stack mode. This property is read-only.
-       * In order to enforce the stack mode, use `stackThreshold` property.
+       * In order to enforce the stack mode, set both `forceOverlay` property and
+       * `stackOverlay` properties to `true`.
```

Comment:
Should this be in the docs for `stackOverlay` instead?

Reply:
Thanks, updated.